.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,32 @@
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+  schedule:
+    - cron: '0 6 * * 1'
+  workflow_dispatch:
+
+jobs:
+  analyze:
+    name: Analyze
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
```

Comment:
Har vi behov for matrix her når det kun er ett språk?

Reply:
Usikker. Jeg fulgte standardoppsettet du får, når du klikker på «Set up this workflow»-knappen.

---

![image](https://user-images.githubusercontent.com/773079/112113720-77326700-8bb7-11eb-848d-1bf22e1aa48f.png)